apps/myaccount/src/components/applications/application-list.tsx

```diff
@@ -131,7 +131,14 @@ export const ApplicationList: FunctionComponent<ApplicationListProps> = (
 
                                         ) }
                                         position="top center"
-                                        content={ app.name }
+                                        content={ (
+                                            <div className="row">
```

Comment:
Can't we use `Grid` component here?
```suggestion
                                            <Grid.Row>
```